src/core/variables.tf

```
@@ -1,15 +1,29 @@
-variable "location" {
-  type    = string
-  default = "westeurope"
-}
+# general
 
 variable "prefix" {
   type    = string
-  default = "pagopa"
+  default = "selc"
+  validation {
+    condition = (
+      length(var.prefix) > 6
```

```suggestion
      length(var.prefix) <= 6
```
my error on template